frontend/src/App.js

```diff
@@ -2,27 +2,34 @@ import React, { Component, Fragment } from 'react';
 import NavBar from './NavBar';
 import Home from './Home';
 
+const LOGGED_IN_STATUS = "logged in";
+
 class App extends Component {
   constructor(props) {
     super(props);
-    this.state = {
-      isLoggedIn: false,
+    this.state = { 
+      isLoggedIn: false, 
+      logURL: "" 
     };
   }
 
-  // Test to see if backend servlet connects to frontend.
-  componentDidMount() {
-    fetch("/api/v1/test-servlet")
-      .then((response) => response.text())
-      .then((text) => {
-        console.log("here is the text from servlet: ", text);
-      });
+  async componentDidMount() {
+    const loginResponse = await fetch("/api/v1/login").then(response => response.text());
+
+    // results[0] is the login status, results[1] is the login or logout URL.
+    const results = loginResponse.split("\n");
+    if (results[0] === LOGGED_IN_STATUS) {
+      this.setState({ isLoggedIn: true });
+    } else {
+      this.setState({ isLoggedIn: false });
```

Comment:
same comment as the `LoginServlet` where a more structure data format is probably preferable.